src/components/App.css

```diff
@@ -1,30 +1,325 @@
-table {
-  width: 60%;
-  margin: 10% 20%;
-  border: 1px solid black;
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+.BookStore-CMS {
+  background-color: #fafafa;
+  height: 90rem;
 }
 
-th {
-  border: 1px solid black;
+header {
+  width: 100%;
+  height: 5.938rem;
+  margin: 0 0 2.375rem 0;
+  background-color: white;
 }
 
-.book-form {
-  width: 70%;
-  margin: 5% 20%;
+.header-nav {
+  display: flex;
```

Comment:
- [ ] It would be better if we implement the CSC DRY principle, there are a few repeating rules that needs to be fix. For example; ‘display: flex’ , ‘ justify-content: space-between’, ‘   font-weight: normal;
   font-stretch: normal;
   font-style: normal;’ etc